src/odbc/src/statement.cpp

```diff
@@ -551,6 +551,18 @@ SqlResult::Type Statement::InternalGetAttribute(int attr, void* buf, SQLINTEGER,
       break;
     }
 
+    case SQL_ROWSET_SIZE: {
+      SQLINTEGER* val = reinterpret_cast<SQLINTEGER*>(buf);
```

Comment:
I think this is a SQLULEN but can't find a reference in the docs.

Reply:
It makes sense to me that this is a `SqlUint`. I have changed this and changed `valueLen` to be set to `SQL_IS_UINTEGER`.